judges/eliminate-ghosts/eliminate-ghosts.rb

```diff
@@ -18,7 +18,7 @@
 
 Fbe.conclude do
   quota_aware
-  on '(and (eq where "github") (exists who) (not (exists stale)))'
+  on '(and (eq where "github") (exists who) (unique who) (not (exists stale)))'
```

Comment:
@Yegorov this will give us only the facts where this user is seen first, while all other facts won't be checked. However, we need _all_ facts to be scanned by this judge.

Reply:
@Yegorov I'm not sure I understand what you suggest exactly

Reply:
@yegor256 the problem with this judge is that he looks at all the facts, although most facts do not require the set `stale` property.
I suggest first to set the `stale` property for only one fact.
And the next step will be adding `stale` property, only to the facts that need it.
Look at my latest commit.